master/buildbot/test/unit/test_www_oauth.py

```diff
@@ -210,6 +216,144 @@ def test_GithubAcceptToken(self):
+            {
+                'data': {
+                    'viewer': {
+                        'organizations': {
+                            'edges': [
+                                {
+                                    'node': {
+                                        'login': 'hello'
+                                    }
+                                },
+                                {
+                                    'node': {
+                                        'login': 'grp'
+                                    }
+                                }
+                            ]
+                        },
+                        'login': 'bar',
+                        'email': 'bar@foo',
+                        'name': 'foo bar'
+                    }
+                }
+            },
+            {
+                'data': {
+                    'hello': {
+                        'teams': {
+                            'edges': [
+                                {
+                                    'node': {
+                                        'name': 'developers',
+                                        'slug': 'develpers'
+                                    }
+                                },
+                                {
+                                    'node': {
+                                        'name': 'contributors',
+                                        'slug': 'contributors'
+                                    }
+                                }
+                            ]
+                        }
+                    },
+                    'grp': {
+                        'teams': {
+                            'edges': [
+                                {
+                                    'node': {
+                                        'name': 'developers',
+                                        'slug': 'develpers'
+                                    }
+                                },
+                                {
+                                    'node': {
+                                        'name': 'contributors',
+                                        'slug': 'contributors'
+                                    }
+                                },
+                                {
+                                    'node': {
+                                        'name': 'committers',
+                                        'slug': 'committers'
+                                    }
+                                },
+                            ]
+                        }
+                    },
+                }
+            }
+        ])
+        res = yield self.githubAuth_v4_teams.acceptToken("TOK3N")
+        self.assertEqual({'email': 'bar@foo',
+                          'username': 'bar',
+                          'groups': ["hello", "grp"],
+                          'teams': [
```

Comment:
You tell me. The idea was a clean separation. If you wish to have `grp` as a role, you'd pass an empty group prefix and we'd end up with a `grp/committers` role too for example.